mapbox/android-docs
# [docs] dr-ui 0.19.3

### Checklist
* [x] Updates dr-ui to 0.19.3
* [x] Add mimic-fn and debounce-fn to `babelInclude` so they will be polyfilled
* [x] Update the staging webhook url `fbtme2z025` -> `evj5gwoa8j`
* [x] Update react and react-dom to version recommended as a peer dependency for search-ui (if necessary)
* [x] Update package.lock with fixes from npm audit fix
* [x] Test IE11 locally
* [ ] Push to staging to confirm build
* [x] Run `npm test` locally to check for broken link warnings


### Screenshot confirmations

![image](https://user-images.githubusercontent.com/2180540/62154508-da7dd000-b2d4-11e9-9f1e-55206d8b52a8.png)

_:+1: IE11 locally_ 

![image](https://user-images.githubusercontent.com/2180540/62154320-73601b80-b2d4-11e9-9fd0-8833280a9224.png)

_:+1: staging webhook_

### To test this PR:

1. npm install && npm start
2. open browserstack in ie11 
3. verify the site loads and there are no errors logged to the console (beside `main` React warning)

Refs: https://github.com/mapbox/documentation/issues/227
